comms/dht/src/envelope.rs

```diff
@@ -110,6 +135,7 @@ pub struct DhtMessageHeader {
     pub network: Network,
     pub flags: DhtMessageFlags,
     pub message_tag: MessageTag,
+    pub expires: Option<DateTime<Utc>>,
```

Comment:
This should probably just be an Epoch, same as we use in the block header

Reply:
Done.